i18n/overrides/en.json

```diff
@@ -9,6 +9,7 @@
 	"miraheze-dberr-again": "Try waiting a few minutes and reloading. For more information on this error please check out our twitter page (https://twitter.com/Miraheze) or contact the system administrators by connecting to #miraheze-sre on irc.libera.chat.",
 	"miraheze-centralauth-groupname": "Miraheze",
 	"miraheze-centralauth-login-error-locked": "{{GENDER:$1|You}} cannot log in because {{GENDER:$1|your}} account is globally locked. Please contact <code>cvt[at]miraheze.org</code> for more information or to appeal.",
+	"miraheze-importtext": "Please export the file from the source wiki using the [[Special:Export|export utility]].\nSave it to your computer and upload it here.<br /> Please note that if the file is large (~ >5MB) you will need to [https://phabricator.miraheze.org/maniphest/task/edit/form/1/ request this on Phabricator] as this cannot be done via a browser",
```

Comment:
The Phabricator link can use internal style interwiki link.

Reply:
Ah true, I guess I didn't think of that